languages/python/sqlalchemy-oso/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: build test test-requirements fmt lint
 build:
 	rm -rf build
-	python setup.py develop
+	pip install -e .[flask]
```

Comment:
@plotnick to test whether these brackets need to be escaped (or the entire `.[flask]` quoted)

Reply:
Seems ok.